### ISSUE TITLE
SSIM 및 평가 방식 관련

### ISSUE THREAD

**Issue by PyewDegul** [LISTatSNU/FastMRI_challenge#5]
안녕하세요..! 열심히 Pytorch 돌리고 있는 참여자 입니다.

오늘 Q&A가 없어 여기에 질문 글 남깁니다..

1. ValLoss와 SSIM 사이의 관계가 어떻게 되는지 궁금합니다. 예를 들어 ValLoss가 0.3이 나온 경우 SSIM=0.7로 이해해도 되는 것인지 궁금합니다.
2. 이미지를 평가할 때 어떤 데이터를 기반으로 평가가 이뤄지는 지 궁금합니다. 지금 주어진 것처럼 brain#.h5만 평가에 사용하는지, 또는 k-space Data도 평가에 이용되는 지 궁금합니다.

감사합니다 :)

**Comment by Juhung** [comment]
Loss 와 SSIM 사이에 관계가 있지는 않습니다.
Reconstruction 된 영상의 품질을 평가하는 척도로써, 하나의 metric 이 그 영상 품질을 모두 대변할 수 없기 때문에
다양한 metric을 써서 비교한다고 생각하면 될 것 같습니다!

**Comment by PyewDegul** [comment]
> Loss 와 SSIM 사이에 관계가 있지는 않습니다.
> Reconstruction 된 영상의 품질을 평가하는 척도로써, 하나의 metric 이 그 영상 품질을 모두 대변할 수 없기 때문에
> 다양한 metric을 써서 비교한다고 생각하면 될 것 같습니다!

답변 감사합니다..! 그렇다면 원 이미지와 recon을 통해 만든 이미지 사이 SSIM을 직접 계산해야 하는 군요..!

**Comment by heewoong98** [comment]
challenge 결과 평가 방식이 SSIM으로만 이루어지는 것이 아니라는 뜻인가요?

**Comment by PyewDegul** [comment]
> challenge 결과 평가 방식이 SSIM으로만 이루어지는 것이 아니라는 뜻인가요?

저도 다시 보니 코드에서 

loss_type = SSIMLoss().to(device=device)

이렇게 선언하고 있어서 약간 헷갈리네요 ㅠㅠㅠ 낼 질문드려봐야 될 것 같습니다!

**Comment by jonghoyi75** [comment]
평가 방식은 Output영상과 Label영상 사이의 SSIM으로 이루이집니다. 자세한 사항은 Overview 4에서 공개가 될 예정입니다. 
LOSS에서 SSIM을 사용하더라도 그 결과와 평가의 결과는 달라질 수 있습니다. 한가지 예로 평가시 머리 외부의 noise 부분의 제외하고 SSIM을 구하도록 할 수도 있습니다.

**Comment by PyewDegul** [comment]
> 평가 방식은 Output영상과 Label영상 사이의 SSIM으로 이루이집니다. 자세한 사항은 Overview 4에서 공개가 될 예정입니다.
> LOSS에서 SSIM을 사용하더라도 그 결과와 평가의 결과는 달라질 수 있습니다. 한가지 예로 평가시 머리 외부의 noise 부분의 제외하고 SSIM을 구하도록 할 수도 있습니다.

답변 감사합니다 :)
